multi-watch/settingmanager.h

```diff
@@ -0,0 +1,23 @@
+#ifndef SETTINGMANAGER_H
+#define SETTINGMANAGER_H
+
+#include <QObject>
```

Comment:
If an `#include` is necessary for your class, move it to `.cpp` file.
Keep the one that can not be moved in `.h` file